src/EditorFeatures/CSharpTest/Formatting/CodeCleanupTests.cs

```diff
@@ -830,10 +830,6 @@ public async Task DoNotApplyFixerIfChangesAreMadeOutsideDocument()
             project = project.AddAnalyzerConfigDocument(".editorconfig", SourceText.From(editorconfigText), filePath: @"z:\\.editorconfig").Project;
             workspace.TryApplyChanges(project.Solution);
 
-            // register this workspace to solution crawler so that analyzer service associate itself with given workspace
-            var incrementalAnalyzerProvider = workspace.ExportProvider.GetExportedValue<IDiagnosticAnalyzerService>() as IIncrementalAnalyzerProvider;
-            incrementalAnalyzerProvider.CreateIncrementalAnalyzer(workspace);
```

Comment:
a lot of these calls were bogus.  they were effectively trying to simulate the solution-crawler starting up and creating the incremental analyzer for a workspace.

we cannot depend on solution crawler for that, so now the underlying system ensures that the incremental analyzers are created on demand.